implementation/jwt-build/src/test/java/io/smallrye/jwt/build/JwtBuildConfigSource.java

```diff
@@ -80,8 +93,10 @@ public void setAudiencePropertyRequired(boolean audiencePropertyRequired) {
 
     @Override
     public Set<String> getPropertyNames() {
-        return new HashSet<>(Arrays.asList("smallrye.jwt.sign.key-location",
-                "smallrye.jwt.encrypt.key-location",
+        return new HashSet<>(Arrays.asList("smallrye.jwt.sign.key.location",
```

Comment:
We can probably static init this (or just init it in the constructor)

Reply:
Hi Roberto @radcortez I did some cleanup of this test config source (with a positive side-effect is that I've got all supported properties now declared in a single place in `JwtBuildUtils`).
I've just stopped short of refactoring `getValue` for it not be implemented in terms of `getProperties` as it is mostly conditional there per different test requirements so it would be also quite messy in `getValue()` :-)